.circleci/config.yml

```
@@ -719,7 +748,7 @@ workflows:
           pg_major: 15
           image_tag: '<< pipeline.parameters.pg15_version >>'
           requires: [build-flaky-15]
-          make: '<< pipeline.parameters.flaky_test_make >>'
+          schedule: '<< pipeline.parameters.flaky_test_schedule >>'
           test: '<< pipeline.parameters.flaky_test >>'
```

I think it's good to keep the old default here. For flaky test debugging you want to run a lot of tests, and you care less about the runtime. For normal flaky test detection that's done in every PR the tradeoff is different .And if you do you can change this manually. 
```suggestion
          runs: 1600
```

Maybe as a parameter though for easy changing at the top of the file, together with `flaky_test`
```
          runs: << pipeline.parameters.flaky_test_runs >>